bcdr/4.1.0/Dockerfile

```diff
@@ -2,14 +2,14 @@ FROM redhat/ubi8-minimal:latest
 
 #Install the Velero and OpenShift client tool alongwith curl, jq and wget
 RUN microdnf update -y && microdnf install -y curl tar gzip jq  && \
-    curl -LO https://github.com/vmware-tanzu/velero/releases/download/v1.9.0/velero-v1.9.0-linux-amd64.tar.gz && \
-    tar -xzf velero-v1.9.0-linux-amd64.tar.gz && \
-    mv velero-v1.9.0-linux-amd64/velero /usr/local/bin/velero && \
-    rm -rf velero-v1.9.0-linux-amd64* && \
-    curl -LO https://github.com/openshift/okd/releases/download/4.10.0-0.okd-2022-03-07-131213/openshift-client-linux-4.10.0-0.okd-2022-03-07-131213.tar.gz && \
-    tar xvf openshift-client-linux-4.10.0-0.okd-2022-03-07-131213.tar.gz && \
-     mv oc /usr/local/bin && \
-    rm -rf openshift-client-linux-4.10.0-0.okd-2022-03-07-131213* && \
+    curl -LO https://github.com/vmware-tanzu/velero/releases/download/v1.10.3/velero-v1.10.3-linux-amd64.tar.gz && \
+    tar -xzf velero-v1.10.3-linux-amd64.tar.gz  && \
+    mv velero-v1.10.3-linux-amd64/velero /usr/local/bin/velero && \
+    rm -rf velero-v1.10.3-linux-amd64* && \
+    curl -LO https://github.com/okd-project/okd/releases/download/4.12.0-0.okd-2023-04-16-041331/openshift-client-linux-4.12.0-0.okd-2023-04-16-041331.tar.gz && \
+    tar xvf openshift-client-linux-4.12.0-0.okd-2023-04-16-041331.tar.gz && \
+    mv oc /usr/local/bin && \
+    rm -rf openshift-client-linux-4.12.0-0.okd-2023-04-16-041331* && \
```

Comment:
How about this instead? (It is newer)

https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable-4.12/
https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable-4.12/openshift-install-linux.tar.gz